packages/@expo/cli/src/prebuild/index.ts

```diff
@@ -15,6 +15,7 @@ export const expoPrebuild: Command = async (argv) => {
       '--yarn': Boolean,
       '--no-install': Boolean,
       '--template': String,
+      '--example': String,
```

Comment:
Prebuild does not support `--example`, create-expo-app [is here](https://github.com/expo/expo-cli/blob/670e3c89c9ad6afb8a803140fce30b3ecf8091e0/packages/create-expo-app/src/cli.ts#L41) and it appears to be documented.

Reply:
Oh sorry, totally wrong location - didn't notice the path of this file

````
packages/@expo/cli/src/prebuild/index.ts
```

Was just going off @amandeepmittal's comment that this was the right file to go with: https://github.com/expo/expo/pull/21463#issuecomment-1454769005

Sorry for the noise, closing!